test/db/analysis/x86_32

```diff
@@ -629,7 +629,7 @@ aaa
 f~?case
 EOF
 EXPECT=<<EOF
-90
+91
```

Comment:
I think this PR is changing this test in a more important way.
Old behaviour:
![Screenshot 2021-01-23 at 16 06 05](https://user-images.githubusercontent.com/562321/105602431-a5512180-5d95-11eb-92af-e045038cda07.png)
New behaviour:
![Screenshot 2021-01-23 at 16 06 16](https://user-images.githubusercontent.com/562321/105602501-aa15d580-5d95-11eb-941c-ebc2c53a12a7.png)
Jump table:
![Screenshot 2021-01-23 at 16 06 42](https://user-images.githubusercontent.com/562321/105602592-b00bb680-5d95-11eb-9566-bf80367e699d.png)

The basic block shown in the first two images is 0x10000192c + 4294965314 (modulo 32bit), which as you can see is the selected case of 1...14, etc., as shown in the old behaviour, not the new one.

Reply:
I am guessing that it's the result if implementing "shift of first case value" support. The question if it's false posivite for shift logic or are the new results  more correct.

Reply:
![image](https://user-images.githubusercontent.com/7101031/105606323-5315fd80-5da1-11eb-8474-42472fc0fefb.png)

The code subtracts 49 from the index before doing lookup in jump table. Seems likely that compiler removed first 49 unused entrie to reduce table size.